testgrid/config/config.yaml

```diff
@@ -1106,6 +1106,15 @@ test_groups:
   gcs_prefix: kopeio-kubernetes-e2e/logs/kubernetes-e2e-upup-aws-ha
 - name: kopeio-kubernetes-e2e-upup-aws
   gcs_prefix: kopeio-kubernetes-e2e/logs/kubernetes-e2e-upup-aws
+# @luxas' multiarch e2e results
+- name: periodic-multi-platform-kubeadm-gce-amd64
+  gcs_prefix: kubernetes-multiarch-e2e-results/logs/kubeadm-luxas-gce-amd64
```

Comment:
Bucket and all files in the bucket needs anonymous read. This url shoul work in an incognito bucket. Compare:
* http://storage.googleapis.com/kubernetes-multiarch-e2e-results
* http://storage.googleapis.com/kubernetes-jenkins/

Reply:
fixed up, now anonymous read enabled